standardresponse.go

```diff
@@ -40,8 +40,8 @@ func (r *StandardResponse) SetBody(bodyBytes []byte) {
 
 // GetBody will be used by gorest core to retrieve the body to be written in
 // the HTTP response writer.
-func (r *StandardResponse) GetBody() []byte {
-	return r.body
+func (r *StandardResponse) GetBody() ([]byte, error) {
```

Comment:
The `GetBody` function is defined this way in order to implement the [`Response`](https://github.com/fredmaggiowski/gorest/blob/master/response.go#L9) interface, otherwise we won't be able to use the `StandardResponse` inside gorest.

Reply:
Yeah but the interface have the error if you check what do you have linked